sagemaker-remote-function/quick_start/environment.yml

```diff
@@ -0,0 +1,7 @@
+name: remote_conda_env
+channels:
+  - defaults
+dependencies:
+  - pandas
```

Comment:
You need to specify the python version explicitly. I think the default one is 3.11 now.

Reply:
+1